src/sentry/models/release.py

```diff
@@ -403,15 +403,14 @@ def set_commits(self, commit_list):
                         author = None
                     elif author_email not in authors:
```

Comment:
not that this is in your change and also not sure if I'm missing something but isn't `authors` always `{}` at this point? wondering if we need this elif/else